pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -168,29 +219,41 @@ Function | Description
 
 For any function you defined as an `export` in your `gulpfile.js`, you can run `npx gulp [function]`.
 
-## Step 5: Initialize your project
-Initialize your project to copy all the necessary image, font, and Javascript assets from the source code.
+For example, if you wanted to have the ability to tell Gulp to compile your Sass, you'd need to first export the `compile` task for your project by adding this line to your `gulpfile.js`:
+
+```
+exports.compile = uswds.compile;
+```
+
+Once exported, you can compile your code by running the following in the command line from your project root:
+
+```
+npx gulp compile
+```
+
+### Step 6: Initialize your project
+
+When your `gulfile.js` is ready, initialize your project to copy all the necessary Sass, image, font, and Javascript assets from the USWDS source code.
 
 Initialize your project and get started by running the following command:
 
-{:.site-terminal}
 ```bash
 npx gulp init
 ```
 
+This command will add all the USWDS assets to the directories you set, add a project Sass entry point, and compile USWDS into CSS. Add this CSS file to the `<head>` of your project HTML.
+
 {: .site-note }
 **Note:** Use `init` only once. The `init` task is meant for initializing the design system on a project. Since it will overwrite project files (like settings files and the Sass entry point), use it sparingly and don't use it for updating the design system on a project, or at any point after you've customized your settings files.
 
 If you receive the error `replaceAll is not a function` when trying to run `npx gulp init`, please verify you are using the version of Node specified in the [.nvmrc file](https://github.com/uswds/uswds/blob/main/.nvmrc) and run the command again.
 
-This command will add all the USWDS assets to the directories you set, add a project Sass entry point, and compile USWDS into CSS. Add this CSS file to the `<head>` of your project HTML.
+### Step 7: Verify successful installation
 
-## Step 6: Verify successful installation
-Any time you want to recompile your CSS, run `npx gulp compileSass` from the command line in your project root.
```

Comment:
Changed this to `npx gulp compile` because that is the name of the task we export in step 2

Reply:
Smart addition!